core/echo_mock.go

```diff
@@ -5,35 +5,36 @@
 package core
 
 import (
+	reflect "reflect"
+
 	gomock "github.com/golang/mock/gomock"
 	echo "github.com/labstack/echo/v4"
-	reflect "reflect"
 )
 
-// MockEchoServer is a mock of EchoServer interface
+// MockEchoServer is a mock of EchoServer interface.
```

Comment:
did you install the latest tools using the makefile? If so, what version is your mockgen on?